sccp/figures/common.py

```diff
@@ -99,7 +101,7 @@ def genFigure():
     print(f"Figure {sys.argv[1]} is done after {time.time() - start} seconds.\n")
 
 
-def plotFactors(factors, data: Pf2X, axs, reorder=tuple(), trim=tuple(), saveGenes=False, row_colors= None):
+def plotFactors(factors, data: Pf2X, axs, reorder=tuple(), trim=tuple(), saveGenes=False, cond_group_labels= None):
```

Comment:
this function is ACTUALLY more generalizable now. both previous versions relied on defining `SLE` as one color and `Healthy` as another

Reply:
Is the input for this labels for each condition?